### PR TITLE
feat(wikipedia): init

### PR DESCRIPTION
**Pull request by GitMuslim**
## 🎉 Theme for Wikipedia  🎉

Wikipedia

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/src/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [catwalk](https://github.com/catppuccin/toolbox#-catwalk).

### CODE REVIEW

**Comment by isabelroses** [comment]
image

this create account page is improperly themed

**Comment by uncenter** [comment]
**Homepage language selector**:
Screenshot 2024-01-17 at 21 44 59 (Arc)

**Overscroll areas above and below page**:
Screenshot 2024-01-17 at 21 46 35 (Arc)

**Peek boxes settings icon**:
Screenshot 2024-01-17 at 21 48 03 (Arc)

**Picture of the day page**:
https://en.wikipedia.org/wiki/Template:POTD/2024-01-17

**Search icon** (MEDIUM/SMALL VIEWPORT WIDTH ONLY):
Screenshot 2024-01-17 at 21 49 30 (Arc)

**TOC button** (MEDIUM/SMALL VIEWPORT WIDTH ONLY):
(see top left white box)
Screenshot 2024-01-17 at 21 50 40 (Arc)

**Note/callout boxes**:
(https://en.wikipedia.org/wiki/Wikipedia:Citation_needed)
Screenshot 2024-01-17 at 21 51 39 (Arc)

File: styles/wikipedia/catppuccin.user.css [hunk 1]
```diff
@@ -0,0 +1,626 @@
...
+    h2 {
+      border-bottom: 1px solid @surface2;
+    }
+    
+    .mw-footer {
+      border-top: 1px solid @surface2;
+    }
+    
+    .bookshelf {
+      border-top: 1px solid @surface0;
```
Comment (uncenter) [inline styles/wikipedia/catppuccin.user.css:192]:
```suggestion
      border-bottom-color: @surface2;
    }
    
    .mw-footer {
      border-top-color: @surface2;
    }
    
    .bookshelf {
      border-top-color: @surface0;
```

File: styles/wikipedia/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,626 @@
...
+      box-shadow: 0 -1px 0 @surface0
+    }
+    
+    .mw-parser-output tr + tr > .navbox-list, .mw-parser-output tr + tr > .navbox-group {
+      border-top: 2px solid @surface0;
```
Comment (uncenter) [inline styles/wikipedia/catppuccin.user.css:197]:
```suggestion
      border-top-color: @surface0;
```

File: styles/wikipedia/catppuccin.user.css [hunk 3]
```diff
@@ -0,0 +1,626 @@
...
+        background-color: @base;
+      }
+    }
+    
+    .vector-feature-zebra-design-enabled .vector-pinnable-header, .vector-pinnable-header, .vector-pinnable-element .vector-menu-heading {
+      border-bottom: 1px solid @surface0;
+    }
+    
+    hr {
+      border-bottom: 1px solid @surface2;
```
Comment (uncenter) [inline styles/wikipedia/catppuccin.user.css:265]:
```suggestion
      border-bottom-color: @surface0;
    }
    
    hr {
      border-bottom-color: @surface2;
```

File: styles/wikipedia/catppuccin.user.css [hunk 4]
```diff
@@ -0,0 +1,626 @@
...
+      color: @text !important;
+      border-top: 1px solid @surface2;
+      border-left: 1px solid @surface2;
+      border-right: 1px solid @surface2;
+      & > figcaption {
+        background-color: @mantle !important;
+        color: @text !important;
+        border-bottom: 1px solid @surface2;
+        border-left: 1px solid @surface2;
+        border-right: 1px solid @surface2;
```
Comment (uncenter) [inline styles/wikipedia/catppuccin.user.css:338]:
```suggestion
      border-top-color: @surface2;
      border-left-color:@surface2;
      border-right-color: @surface2;
      & > figcaption {
        background-color: @mantle !important;
        color: @text !important;
        border-bottom-color: @surface2;
        border-left-color: @surface2;
        border-right-color: @surface2;
```

File: styles/wikipedia/catppuccin.user.css [hunk 5]
```diff
@@ -0,0 +1,626 @@
...
+    .mw-parser-output .navbox-title {
+      background-color: fade(@color, 20%) !important;
+    }
+    
+    .mw-parser-output .navbox-group {
+      background-color: fade(@color, 10%) !important;
+    }
+    
+    .catlinks li {
+      border-left: 1px solid @surface2;
```
Comment (uncenter) [inline styles/wikipedia/catppuccin.user.css:370]:
```suggestion
      border-left-color: @surface2;
```

**Comment by isabelroses** [comment]
Some tables have bad contrast:
image

When performing a search the background appears white:
image

When looking around i could only find these pages (given they are not visited that much):
- https://en.wikipedia.org/wiki/Talk:Main_Page
- https://en.wikipedia.org/w/index.php?title=Main_Page&action=history
- https://en.wikipedia.org/w/index.php?title=Main_Page&action=edit
- https://en.wikipedia.org/wiki/Talk:Main_Page#Main_Page_error_reports

one example of what the pages might look like:
image

Other then that, I cannot find anything to breaking.

**Review by isabelroses — CHANGES_REQUESTED** [review]
see prior comments

**Comment by GenShibe** [comment]
image
image
image
a bunch of table stuff needs to be themed

**Comment by GitMuslim** [comment]
@isabelroses @GenShibe fixed all of the above

**Review by isabelroses — CHANGES_REQUESTED** [review]
The last blocker I can realisitcly see is the contrast here
image
[link to reproduce](https://en.wikipedia.org/wiki/Second_War_of_Scottish_Independence)

**Comment by GitMuslim** [comment]
done

**Pull request merged**